File: backend/GestionSetlistApp/Models/MorceauSetlist.cs
namespace GestionSetlistApp.Models
{
    public class MorceauSetlist
    {
        public required int MorceauId { get; set; }
        public required Morceau? Morceau { get; set; }
        public required int SetlistId { get; set; }
        public required Setlist Setlist { get; set; }
        public required int Position { get; set; }
    }
}

File: frontend/gestion-setlist/src/app/models/morceauSetlist.ts
import { Morceau } from "./morceau";
import { Setlist } from "./setlist";

export interface MorceauSetlist {
    morceauId: number;
    morceau: Morceau;
    setlistId: number;
    setlist: Setlist;
    position: number;
}
